src/test/java/org/opentripplanner/apis/gtfs/GraphQLFormattingTest.java

```diff
@@ -0,0 +1,24 @@
+package org.opentripplanner.apis.gtfs;
+
+import static org.opentripplanner.framework.io.FileUtils.assertFileEquals;
+import static org.opentripplanner.framework.io.FileUtils.readFile;
+import static org.opentripplanner.framework.io.FileUtils.writeFile;
+
+import graphql.schema.idl.SchemaPrinter;
+import java.io.File;
+import org.junit.jupiter.api.Test;
+
+public class GraphQLFormattingTest {
+
+  public static final File SCHEMA_FILE = new File(
```

Comment:
You can use the `ResourceLoader` for this.

Reply:
Ok thanks, I thought there was a better way to do this. I Copied the approach I used from the transmodel test.